includes/donors/class-give-donors-query.php

```diff
@@ -621,11 +621,21 @@ public function date_filter_pre() {
 		$date_query = array();
 
 		if ( ! empty ( $this->args['start_date'] ) ) {
-			$date_query['after'] = give_get_formatted_date( $this->args['start_date'] );
+			$date_query['after'] = date(
```

Comment:
Lots more code, can this be done in a dryer way?

Reply:
@DevinWalker I did two tings here:

1. since `give_get_formatted_date ` does not work well with translated localized date strings, so we start using `date` fn to convert the date to `MySQL` date format which is standard.

2. Added support for the timestamp in `start_date` and `end_date` query param.